pkg/loop/internal/pb/chain_reader.proto

```diff
@@ -11,23 +11,23 @@ service ChainReader {
   rpc GetLatestValue (GetLatestValueRequest) returns (GetLatestValueReply) {}
   rpc QueryKey(QueryKeyRequest) returns (QueryKeyReply) {}
   rpc Bind(BindRequest) returns (google.protobuf.Empty) {}
+  rpc Unbind(BindRequest) returns (google.protobuf.Empty) {}
 }
 
 // GetLatestValueRequest has arguments for [github.com/smartcontractkit/chainlink-common/pkg/types.ChainReader.GetLatestValue].
 message GetLatestValueRequest {
-  string contract_name = 1;
-  string method = 2;
-  VersionedBytes params = 3;
+  BoundContract contract = 1;
+  VersionedBytes params = 2;
 }
 
 // QueryKeyRequest has arguments for [github.com/smartcontractkit/chainlink-common/pkg/types.ChainReader.QueryKey].
 message QueryKeyRequest {
-  string contract_name = 1;
+  BoundContract contract = 1;
   QueryKeyFilter filter = 2;
   LimitAndSort limit_and_sort = 3;
 }
 
-// BindRequest has arguments for [github.com/smartcontractkit/chainlink-common/pkg/types.ChainReader.Bind].
+// BindRequest has arguments for [github.com/smartcontractkit/chainlink-common/pkg/types.ChainReader.Bind/Unbind].
 message BindRequest {
```

Comment:
Probably better to have BindRequest and UnBindRequest separately

Reply:
I started doing that but left it as the same to keep the interface simpler. I can make them separate to ensure extensibility instead.